include/rmm/detail/dynamic_load_runtime.hpp

```diff
@@ -116,6 +121,28 @@ struct async_alloc {
     return runtime_supports_pool and driver_supports_pool;
   }
 
+  /**
+   * @brief Check whether the specified `cudaMemAllocationHandleType` is supported on the present
+   * CUDA driver/runtime version.
+   *
+   * @note This query was introduced in CUDA 11.3 so on CUDA 11.2 this will only return true for
+   * `cudaMemHandleTypeNone`.
+   *
+   * @param handle_type An IPC export handle type to check for support.
+   * @return true if supported
+   * @return false if unsupported
+   */
+  static bool is_export_handle_type_supported(cudaMemAllocationHandleType handle_type)
+  {
+    int supported_handle_types_bitmask{};
+#if CUDART_VERSION >= 11030  // 11.3 introduced cudaDevAttrMemoryPoolSupportedHandleTypes
+    cudaDeviceGetAttribute(&supported_handle_types_bitmask,
+                           cudaDevAttrMemoryPoolSupportedHandleTypes,
+                           rmm::detail::current_device().value());
+#endif
+    return (supported_handle_types_bitmask & handle_type) == handle_type;
```

Comment:
if `CUDART_VERSION` <11030 this function will use `supported_handle_types_bitmask` uninitialized . I'm not sure if the best option is to use an `#else` or initialize it to a value to match the desired return value.

Reply:
It is default initialized (the curly braces on line 137) to 0, indicating that no handle types are supported, which is what we want. `0 & X == X` iff `X == 0` since `&`ing with 0 will zero out all the bits.

Reply:
My mistake. Thanks for the extra information that a 0 unit is desired here.